Validation/RecoTrack/python/PostProcessorTracker_cfi.py

```diff
@@ -348,11 +349,20 @@ def _addNoFlow(module):
 displacedTrackValidation.toReplaceWith(postProcessorTrack,postProcessorTrackDisplaced)
 displacedTrackValidation.toReplaceWith(postProcessorTrackSummary,postProcessorTrackSummaryDisplaced)
 
+from Configuration.ProcessModifiers.pp_on_AA_cff import pp_on_AA
+
+_defaultSubdirsHIon =  _defaultSubdirs + ["Tracking/HIPixelTrack/*"]
+_defaultSubdirsSummaryHIon = [e.replace("/*","") for e in _defaultSubdirsHIon]
+
+pp_on_AA.toModify(postProcessorTrack,subDirs = _defaultSubdirsHIon)
+pp_on_AA.toModify(postProcessorTrack2D,subDirs = _defaultSubdirsHIon)
+pp_on_AA.toModify(postProcessorTrackSummary,subDirs = _defaultSubdirsHIon)
+
 
 postProcessorTrackTrackingOnly = postProcessorTrack.clone()
-postProcessorTrackTrackingOnly.subDirs.extend(["Tracking/TrackBHadron/*", "Tracking/TrackSeeding/*", "Tracking/PixelTrack/*", "Tracking/PixelTrackFromPV/*", "Tracking/PixelTrackFromPVAllTP/*", "Tracking/PixelTrackBHadron/*"])
+postProcessorTrackTrackingOnly.subDirs.extend(["Tracking/TrackBHadron/*", "Tracking/TrackSeeding/*", "Tracking/HIPixelTrack/*", "Tracking/PixelTrack/*", "Tracking/PixelTrackFromPV/*", "Tracking/PixelTrackFromPVAllTP/*", "Tracking/PixelTrackBHadron/*"])
 postProcessorTrackSummaryTrackingOnly = postProcessorTrackSummary.clone()
-postProcessorTrackSummaryTrackingOnly.subDirs.extend(["Tracking/TrackBHadron", "Tracking/TrackSeeding", "Tracking/PixelTrack", "Tracking/PixelTrackFromPV", "Tracking/PixelTrackFromPVAllTP", "Tracking/PixelTrackBHadron"])
+postProcessorTrackSummaryTrackingOnly.subDirs.extend(["Tracking/TrackBHadron", "Tracking/TrackSeeding", "Tracking/HIPixelTrack", "Tracking/PixelTrack", "Tracking/PixelTrackFromPV", "Tracking/PixelTrackFromPVAllTP", "Tracking/PixelTrackBHadron"])
```

Comment:
is it a problem to add the `"Tracking/HIPixelTrack"` folder also for the non-HI workflows ?

Reply:
I will produce more warnings for harvesting (alas logs are already full of them.

Reply:
I think I can "protect" this with the hion era.